runtime/common/src/lib.rs

```diff
@@ -234,6 +234,8 @@ pub type CouncilInstance = pallet_collective::Instance1;
 pub type TechnicalCommitteeInstance = pallet_collective::Instance2;
 pub type CouncilMembershipInstance = pallet_membership::Instance1;
 pub type TechnicalCommitteeMembershipInstance = pallet_membership::Instance2;
+pub type DeveloperCommitteeInstance = pallet_collective::Instance3;
+pub type DeveloperCommitteeMembershipInstance = pallet_membership::Instance3;
```

Comment:
Being picky: I'm wondering why not follow the order of declaration

Reply:
Updated to follow the order ✅